samples/drivers/led/led_strip/boards/nucleo_c071rb.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021, STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/led/led.h>
+
+&spi1 {
+	/* Use pin PA7, labeled as D11/MOSI on the C071RB */
+	pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5
+		     &spi1_miso_pa6 &spi1_mosi_pa7>;
```

Comment:
Drop `spi1_nss_pa4` and `spi1_miso_pa6`, you are not using them and they are free for other functions, `sck` is needed though even though the ws2812 protocol does not use it, but the STM32 SPI controller refuses to do anything if the clock pin is not mapped, since you are doing a sample, can you add a comment about it?